apps/modernization-ui/src/alert/useAlert.tsx

```diff
@@ -24,7 +24,7 @@ type AlertProviderProps = {
     children: ReactNode;
 };
 
-function AlertProvider({ duration = 5000, children }: AlertProviderProps) {
+function AlertProvider({ duration = 5000 * 5000, children }: AlertProviderProps) {
```

Comment:
creeping changes 👻

Reply:
Ugh, fixed.  thought i deleted that. keeps coming back to haunt me lol